packages/less/test/lessc-alpha.mjs

```
@@ -0,0 +1,246 @@
+    assert.equal(file.code, 0, file.stderr);
+    assert.match(file.stdout, /^lessc: wrote .+output\.css\n$/);
+    assert.equal(file.stderr, '');
+    const css = await readFile(output, 'utf8');
+    assert.match(css, /\.from-import\s*\{[\s\S]*color:\s*green;/,
+        'file compilation resolves a sibling import through the CLI');
+    assert.match(css, /\.from-file\s*\{[\s\S]*width:\s*2;/);
+
+    const failure = await runLessc([broken]);
+    assert.equal(failure.code, 1, 'a Less error is a failing lessc process');
+    assert.equal(failure.stdout, '');
+    assert.match(failure.stderr, /\x1B\[[0-?]*[ -/]*m/,
```

ESLint

[error] Unexpected control character(s) in regular expression: \x1b.

(no-control-regex)